packages/manager/modules/navbar/src/index.less

```diff
@@ -158,8 +158,10 @@ ovh-manager-navbar {
   }
 
   .ovh-sidebar-wrapper {
-    position: absolute;
+    position: fixed;
+    top: 2.75rem; /* Navbar height */
```

Comment:
There is no variable which stores this height into ui-kit maybe ?

Reply:
Probably, I'll take a look

Reply:
You're right, nice catch ! Updated ;)